thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/wrapper/AnomalyDetectorWrapper.java

```diff
@@ -164,7 +169,7 @@ public DetectionPipelineResult run() throws Exception {
       anomaly.setDimensions(DetectionUtils.toFilterMap(this.metricEntity.getFilters()));
       anomaly.getProperties().put(PROP_DETECTOR_COMPONENT_NAME, this.detectorName);
     }
-    return new DetectionPipelineResult(anomalies, this.getLastTimeStamp());
+    return new DetectionPipelineResult(anomalies, predictions, this.getLastTimeStamp());
```

Comment:
In the merger, dimension explorer, and anomaly filter wrapper need to handle this predicted time series too.

Reply:
Why merger needs to handle this? Merger's input is anomalies only.